modules/distribution-zones/src/testFixtures/java/org/apache/ignite/internal/distributionzones/DistributionZonesTestUtil.java

```diff
@@ -313,17 +313,17 @@ public static void deployWatchesAndUpdateMetaStorageRevision(MetaStorageManager
             throws NodeStoppingException, InterruptedException {
         // Watches are deployed before distributionZoneManager start in order to update Meta Storage revision before
         // distributionZoneManager's recovery.
-        metaStorageManager.deployWatches();
+        var deployWatchesFut = metaStorageManager.deployWatches();
```

Comment:
AFAIK it's not valid to use var here.

Reply:
This is a test classe. I think it is possible for test purposes.